easybuild/easyblocks/generic/intelbase.py

```diff
@@ -105,7 +105,7 @@ def extra_options(extra_vars=None):
             # disables TMP_PATH env and command line option
             'usetmppath': [False, "Use temporary path for installation", CUSTOM],
             'm32': [False, "Enable 32-bit toolchain", CUSTOM],
-            'components': [[], "List of components to install", CUSTOM],
+            'components': [None, "List of components to install", CUSTOM],
```

Comment:
a list as a default value is evil, because it's mutable

the help message makes it very clear what it should be?

also, with the work on type checking being done, we can specify the expected type here at some point (not yet though, needs more work in framework)